bob/mkosi.extra/usr/bin/init-firewall.sh

```diff
@@ -162,6 +168,15 @@ iptables -A $CHAIN_ALWAYS_ON_OUT -p udp --dport $NTP_PORT \
 iptables -A $CHAIN_ALWAYS_ON_OUT -p tcp -d $TITAN_BUILDER_IP --dport $TITAN_BUNDLE_PORT_HTTPS \
     -m conntrack --ctstate NEW -j ACCEPT
 
+# Flashbots bundle endpoints (always on)
+# Security note: This is a side channel.
+# While the operator will not be able to see the content of the packets, 
+# they can observe the presence or absence of packets.
+iptables -A $CHAIN_ALWAYS_ON_OUT -p tcp -d $FLASHBOTS_BUNDLE_IP1 --dport $HTTPS_PORT \
+    -m conntrack --ctstate NEW -j ACCEPT
+iptables -A $CHAIN_ALWAYS_ON_OUT -p tcp -d $FLASHBOTS_BUNDLE_IP2 --dport $HTTPS_PORT \
+    -m conntrack --ctstate NEW -j ACCEPT
```

Comment:
We can also merge here

```suggestion
iptables -A $CHAIN_ALWAYS_ON_OUT -p tcp -d $FLASHBOTS_BUNDLE_IP1,$FLASHBOTS_BUNDLE_IP2 --dport $HTTPS_PORT \
    -m conntrack --ctstate NEW -j ACCEPT
```